vendor/braintree-web/data-collector/fraudnet.js

```diff
@@ -77,7 +80,11 @@ function _createThirdPartyBlock() {
       this.domain = dom;
     }
     js.id = 'js-iframe-async';
-    js.src = scriptBaseURL + 'fb-all-prod.pp.min.js';
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
Why not default to prod and use a custom script url if an env variable is passed?